juniper_official/routing/check-te-rsvp-global-errors.rule

```diff
@@ -468,28 +772,87 @@
                                     }
                                 }
                                 platforms MX240 {
+                                    releases 20.1R1 {
+                                        sensors lsp-updated;				
+                                        release-support min-supported-release;
+                                    }								
                                     releases 17.4R1 {
                                         release-support min-supported-release;
                                     }
                                 }
                                 platforms MX480 {
+                                    releases 20.1R1 {
+                                        sensors lsp-updated;				
+                                        release-support min-supported-release;
+                                    }								
                                     releases 17.4R1 {
                                         release-support min-supported-release;
                                     }
                                 }
                                 platforms MX960 {
+                                    releases 20.1R1 {
+                                        sensors lsp-updated;				
+                                        release-support min-supported-release;
+                                    }								
                                     releases 17.4R1 {
                                         release-support min-supported-release;
                                     }
                                 }
+                                platforms MX304 {
+                                    releases 20.1R1 {
+                                        sensors lsp-updated;				
+                                        release-support min-supported-release;
+                                    }
+                                }
+                                platforms MX204 {
+                                    releases 20.1R1 {
```

Comment:
Please check the versions. This seems to be an old release. We will have to put all relevant versions if the fix went into multiple versions